examples/ITK_Example20_PixelTypes.ipynb

```diff
@@ -0,0 +1,325 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    intend = 30
</pre><p>intend or indent?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/ITKElastix/pull/224/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Thanks for reviewing! I fixed the typo.